### PR TITLE
make python3 tests pass

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

**Comment by wdpypere** [comment]
@boegel this one also needs the prospector locked to lower than 1.3.

@kwaegema this repo is a prereq for vsc-backup. Can you check I did not make any crazy mistakes?

**Comment by wdpypere** [comment]
@boegel I'm quite confused, with the latest vsc-install I get:
```
  File "/var/lib/jenkins/jobs/hpcugent_github.com/jobs/vsc-zk/branches/PR-21/workspace/.tox/py27/lib/python2.7/site-packages/vsc_install-0.15.13-py2.7.egg/vsc/install/ci.py", line 170, in parse_vsc_ci_cfg

    except (configparser.NoSectionError, configparser.ParsingError) as err:

AttributeError: 'module' object has no attribute 'NoSectionError'
```

**Comment by wdpypere** [comment]
ah, I get it now. this lib has a configparser.py which gets picked up. nasty.

**Comment by wdpypere** [comment]
@boegel @kwaegema tests are green!

File: setup.py [hunk 1]
```diff
@@ -31,16 +31,16 @@
 @author: Stijn De Weirdt (Ghent University)
 """
 import vsc.install.shared_setup as shared_setup
-from vsc.install.shared_setup import ag, kw, sdw
+from vsc.install.shared_setup import kw, sdw
 
 PACKAGE = {
-    'version': '0.9.9',
+    'version': '0.9.10',
```
Comment (boegel) [inline setup.py:37]: Perhaps a major version bump because of the `vsc.zk.zkconfigparser` rename?

**Comment by wdpypere** [comment]
@boegel major version bump done

File: setup.py [hunk 2]
```diff
@@ -31,16 +31,16 @@
...
-from vsc.install.shared_setup import ag, kw, sdw
+from vsc.install.shared_setup import kw, sdw
 
 PACKAGE = {
-    'version': '0.9.9',
+    'version': '1.0.0',
     'author': [sdw, kw],
     'maintainer': [sdw, kw],
     'install_requires': [
         'vsc-base >= 1.6.7',
```
Comment (boegel) [inline setup.py:41]: @wdpypere This needs a bump too, since you're importing from `vsc.utils.py2vs3`

Comment (wdpypere) [reply setup.py:41]: it says `vsc.utils` and I bumped `vsc-utils`?

**Comment by wdpypere** [comment]
@boegel this is a dep for vsc-backup, that is why I'm trying to fix this one first, because I can't test vsc-backup without an updated vsc-zk.

File: setup.py (hunk 2 above)
Comment (boegel) [reply setup.py:41]: `vsc.utils.py2vs3` is in `vsc-base` :)

File: lib/vsc/zk/rsync/source.py [hunk 3]
```diff
@@ -406,8 +408,10 @@ def rsync_path(self, path):
         """ start rsync session for given path and destination, returns true if successful """
         if not path:
             self.log.raiseException('Empty path given!')
-        elif not isinstance(path, basestring):
+            return None
+        elif not is_string(path):
             self.log.raiseException('Invalid path: %s !' % path)
+            return None
```
Comment (kwaegema) [inline lib/vsc/zk/rsync/source.py:414]: also needed when exception is raised?

Comment (kwaegema) [reply lib/vsc/zk/rsync/source.py:414]: nvm, can't return too much i guess :stuck_out_tongue:

Comment (wdpypere) [reply lib/vsc/zk/rsync/source.py:414]: yes, because pylint does not recognise self.log.raiseException as an exception.

**Comment by kwaegema** [comment]
@wdpypere @boegel LGTM, thanks!!

File: setup.py (hunk 2 above)
Comment (wdpypere) [reply setup.py:41]: after all these years that still makes no sense to me whatsoever

**Pull request merged**